100-level-AsyncAPI-Intro/step5.md

```diff
@@ -0,0 +1,20 @@
+In this intergalactic journey, you explored AsyncAPI, a powerful specification for crafting asynchronous APIs. The first module discusses what AsyncAPI is and why to use AsyncAPI, exploring real-world AsyncAPI use cases.
+
+Building on the basics, module two transitions into understanding Event-driven Architecture (EDA), showcasing how AsyncAPI fuels the design of both EDA and microservices.
+
+The third module explores the AsyncAPI specification. You learn about the structure of AsyncAPI and its different components, such as info, channels, components, and tags. You also learn to define operations, messages, and schemas, and the request/reply pattern.
+
+The fourth module focuses on hands-on experience with exercises for writing AsyncAPI code and documents from scratch. It also introduces AsyncAPI Studio for validating your documents.
+
+## AsyncAPI tools
+Ready to explore more of the AsyncAPI world? Check out the AsyncAPI tool ecosystem!
+
+- AsyncAPI Studio helps you design and create a blueprint for your asynchronous API. 
+- AsyncAPI CLI (Command-Line Interface) helps you run AsyncAPI using text commands in a terminal.
+- AsyncAPI Generator helps you plan and build the initial structure of your project. 
+- And more!
```

Comment:
Should we add links?

Reply:
Should i link it to different GitHub repos or documentation?

Reply:
Yes, @kakabisht, we should add links to the tools. Let's link to the tool repository because that is a better user experience. (unless someone else thinks differently? open to suggestions)